app/assets/javascripts/live_search.js

```diff
@@ -23,7 +23,9 @@
     this.$sortBlock = options.$results.querySelector('#js-sort-options')
     this.$paginationBlock = options.$results.querySelector('#js-pagination')
     this.action = this.$form.getAttribute('action') + '.json'
-    this.$atomAutodiscoveryLink = options.$atomAutodiscoveryLink
+    if (options.$atomAutodiscoveryLink) {
```

Comment:
(Just a request for clarification) where does options.$atomAutodiscoverLink get set?

Reply:
Good question, it gets set in application.js - https://github.com/alphagov/finder-frontend/blob/main/app/assets/javascripts/application.js#L37

I've not come across this approach before in our codebase and there is a comment in this file saying this is not advisable - https://github.com/alphagov/finder-frontend/blob/main/app/assets/javascripts/application.js#L7-L8

It's likely possible to clean this up with some refactoring I imagine.

Reply:
Oh, I see - options is just a sort of hold-all for all the named parameters passed in when the GOVUK.LiveSearch is created. Cool.